src/Pos/Genesis.hs

```diff
@@ -107,19 +109,19 @@ bitcoinDistribution20 :: [Coin]
 bitcoinDistribution20 = map mkCoin
     [200,163,120,105,78,76,57,50,46,31,26,13,11,11,7,4,2,0,0,0]
 
-stakeDistribution :: StakeDistribution -> [Coin]
+stakeDistribution :: StakeDistribution -> [(Coin, [(StakeholderId, Coin)])]
```

Comment:
`(Coin, [(StakeholderId, Coin)])` is not a good type :(
I'm for custom data types in such cases. I'm got tired of list of tuples of lists already in `rscoin`.